server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/delivery/SieveIntegrationTest.java

```diff
@@ -53,11 +53,11 @@
     public static final String LOCAL_PART = "receiver";
     public static final String RECEIVER_DOMAIN_COM = LOCAL_PART + "@domain.com";
 
-    public static DateTimeFormatter formatter = DateTimeFormat.forPattern("yyyy-mm-dd HH:mm:ss");
-    public static final DateTime DATE_CLOSE = formatter.parseDateTime("2016-01-16 00:00:00");
-    public static final DateTime DATE_DEFAULT = formatter.parseDateTime("2016-01-14 00:00:00");
-    public static final DateTime DATE_NEW = formatter.parseDateTime("2016-01-18 00:00:00");
-    public static final DateTime DATE_OLD = formatter.parseDateTime("2011-01-18 00:00:00");
+    public static DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss").withZone(ZoneId.systemDefault());
+    public static final ZonedDateTime DATE_CLOSE = ZonedDateTime.parse("2016-01-16 00:00:00", formatter);
```

Comment:
simply changing the String value would allow to drop the formatter

Reply:
Don't know why but it make some tests fail